api/Window.json

```
@@ -2019,6 +2019,43 @@
           }
         }
       },
+      "getDigitalGoodsService": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/digital-goods/#getdigitalgoodsservice-method",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Can you confirm it shipped in Desktop, too? Per https://chromestatus.com/feature/5339955595313152 it is in Chrome Desktop 100.

```suggestion
              "version_added": "100"
```

---

Chrome 100 isn't out yet, but I cannot confirm support in Chrome 100 Beta on either Windows or macOS, so I think this may be a case of the feature planned to ship, reverted, but then ChromeStatus didn't get updated.

---

hm 109 is the current release, so 100 is out for quite some time now.

---

Oh I'm dumb, I was thinking "110", not "100" (and I even wrote "100" too)...  🤦 

Well, I guess that confirms even more that it hasn't shipped in Desktop yet...!

---

Yeah, I guess chromestatus is wrong. I don't see it working either.